tests/core/full_node/test_hint_management.py

```diff
@@ -97,33 +101,38 @@ async def test_lookup_coin_ids(bt: BlockTools, empty_blockchain: Blockchain) ->
     scs = StateChangeSummary(br, uint32(0), [], npc_res, rewards)
 
     # Removal ID and addition PH
-    coin_subscriptions = {coin_ids[1]: {bytes32(b"2" * 32)}}
-    ph_subscriptions = {phs[4]: {bytes32(b"3" * 32)}}
+    has_coin_sub = lambda c: c == coin_ids[1]  # noqa
```

Comment:
I suggested that maybe `get_hints_subscription_coin_ids()` should be a method.  Not sure if Arvid got to checking out that option quite yet.

Reply:
yeah, it sounds like it could be a good idea, however I'm hesitant to explore that in this PR because:
1. I'd like to stay focused factoring out the subscription data structure, and pulling this function in as a member would significantly increase the size of the patch (the test would have to be updated)
2. It's not obvious that it's better to make it a member. As a free function it's much simpler to unit test.